lstchain/data/lstchain_standard_config.json

```diff
@@ -90,70 +85,95 @@
     "reco_disp_dx",
     "reco_disp_dy"
   ],
-
-  "allowed_tels": [1, 2, 3, 4],
+  "allowed_tels": [
+    1,
+    2,
+    3,
+    4
+  ],
   "max_events": null,
   "custom_calibration": false,
   "write_pe_image": false,
   "image_extractor": "LocalPeakWindowSum",
-  "image_extractor_for_muons": "GlobalPeakWindowSum",
-  "image_extractor_config": {},
+  "image_extractor_config": {
```

Comment:
Why are both configs given here if only `LocalPeakWindowSum` is used?

Reply:
Not need, may (or not) be more clear to be exhaustive on the possible configs, for example if the user wants to switch the `image_extractor`, the parameters are set.
Feedback welcome.

Reply:
`--help-all` should show all possible settings for tools.

Reply:
> `--help-all` should show all possible settings for tools.

Nice.
For that we would need to make Tools out of all scripts.

Reply:
I'd be happy to limit the standard config to only the necessary one if you think it's clearer (I do) - it wasn't, so I kept it this way.

Reply:
@vuillaut that is needed anyways to get to a consistent api and to be able to properly use the configuration system. Currently, this all fells really hacked together around some of the limitation the ctapipe config system has (or had, a lot are fixed in 0.8)